website/src/pages/en/token-api/token-api-faq.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Token API FAQ
+---
+
+## Table of Contents
+
+- [Basic Topics](#basic-topics)
+- [MCP / LLM / AI Topics](#mcp--llm--ai-topics)
+- [Advanced Topics](#advanced-topics)
+  - [Authentication & Access](#authentication--access)
+  - [Network Support](#network-support)
+  - [Query Parameters & Pagination](#query-parameters--pagination)
+  - [Response Format & Data Handling](#response-format--data-handling)
+  - [Common Issues & Troubleshooting](#common-issues--troubleshooting)
+  - [Integration & Usage](#integration--usage)
+
+## General
+
+### What blockchains does the Token API support?
+
+Currently, the Token API supports Ethereum, Binance Smart Chain (BSC), Polygon, Optimism, Base, and Arbitrum One.
+
+### Why isn't my API key from thegraph.market working?
+
+Make sure you are using the Access Token, not the API key. You can generate an access token from the dashboard on thegraph.market, using the dropdown menu next to each API key.
+
+### How current is the data provided by the API relative to the blockchain?
+
+The API provides data up to the Last Irreversible Block (LIB).
+
+### How do I authenticate requests to the Token API?
+
+Authentication is managed via JWT tokens obtained through thegraph.market. JWT tokens issued by Pinax are also supported.
```

Comment:
No explanation of what Pinax is or how/why they issue JWT tokens; maybe we should just omit that sentence?